Sources/CloudyKit/String+Components.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  String+Components.swift
+//
+//
+//  Created by Diego Trevisan on 11.12.23.
+//
+
+import Foundation
+
+public extension String {
```

Comment:
Instead of adding this function could the separation logic just include spaces around the separator?

For example: ```let inSplits = self.predicateFormat.components(separatedBy: " IN ")```